### PR TITLE
Milestone4pr

### PR DESCRIPTION
**Pull request by LadyEr0n**

### CODE REVIEW

**Review by Chaoskil — APPROVED** [review]
Looks all good to me, the code is solid and follows the tutorials to the letter.

File: Assets/Scripts/Door.cs [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+    [Tooltip("Play this audio clip when the player interacts with a locked door without owning the key.")]
+    [SerializeField]
+    private AudioClip lockedAudioClip;
+
+    [Tooltip("Play this audio clip when the player opens the door.")]
+    [SerializeField]
+    private AudioClip openAudioClip;
+
+    // public override string DisplayText => isLocked ? lockedDisplayText : base.DisplayText;
```
Comment (dantogno) [inline Assets/Scripts/Door.cs:28]: remove commented out code

File: Assets/Scripts/Door.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            if (isLocked)
+                toReturn = HasKey ? $"Use {key.ObjectName}" : lockedDisplayText;
+            else
+                toReturn = base.DisplayText;
+
+            return toReturn;
+        }
+    }
+
+
```
Comment (dantogno) [inline Assets/Scripts/Door.cs:45]: remove extra lines

File: Assets/Scripts/InventoryObject.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        renderer = GetComponent<Renderer>();
+        collider = GetComponent<Collider>();
+    }
+
+    public InventoryObject()
+    {
+        displayText = $"Take {objectName}";
+    }
+
+
```
Comment (dantogno) [inline Assets/Scripts/InventoryObject.cs:38]: remove extra lines

**Pull request merged**